addons/mail/models/mail_thread.py

```diff
@@ -3069,12 +3072,16 @@ def _message_auto_subscribe_notify(self, partner_ids, template):
             }
             assignation_msg = self.env['ir.qweb']._render(template, values, minimal_qcontext=True)
             assignation_msg = self.env['mail.render.mixin']._replace_local_links(assignation_msg)
+            subtitles = [record.display_name]
+            if 'date_deadline' in record and record.date_deadline:
```

Comment:
Mmmh, seems like a hack to me. Not sure what is the purpose, but I don't like that kind of detail hidden here. Can't we get rid of it ? Or find a more generic or model-based approach ?